src/main/java/com/example/bcsd/Repository/ArticleRepository.java

```diff
@@ -0,0 +1,90 @@
+package com.example.bcsd.Repository;
+
+import com.example.bcsd.Model.Article;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class ArticleRepository {
```

Comment:
```suggestion
@Repository
@Transactional(readOnly = true)
public class ArticleRepository {
```
보고서에 남긴 리뷰처럼 이렇게 클래스 단에 달아놓고 사용할 수 있습니다!

Reply:
팁 감사합니다 ㅎㅎ 아직 저의 서치에 부족한 부분이 많은 것 같습니다...